app/components/header_component/view.html.erb

```diff
@@ -1,6 +1,7 @@
 <% if @current_user %>
   <%= govuk_header navigation_classes: ["govuk-header__navigation--end"] do |header|
     header.product_name(name: t("header.product_name"))
+    header.navigation_item(text: t('users.menu'), href: users_path, active: false) if @current_user.super_admin?
```

Comment:
I think it would be better if we passed this into the component itself. It will make unit testing the component a lot easier because we wouldn't need to create a user record and mock out these method. we should call the component and pass in a boolean value. In the view, when we call the component, we would use `policy(@current_user).can_manage_user?` which will use the policy to control whether the user can see the link or not. 

When we come to phase 2, we can update the policy and everything should still carry on working. This is the whole point of having a centralised permission policy pattern in that you can go to one place to edit policies that may affect various parts of the system.

Reply:
Actually, could we maybe pair on this component? I think we should try remove the instance vars being using and `attr_accessors` with the same var names. the reason I'm saying this is that when I first reviewed this I thought `@current_user` would be the exact same object as `@current_user` which we using all over the app and so it wasn't clear that we were actually passing in the `@current_user` and then setting another instance var with exactly the same name. very confusing to follow without viewing the components ruby class and where its being called from.